src/components/NavBar.css

```diff
@@ -0,0 +1,25 @@
+nav {
+  position: fixed;
+  box-sizing: border-box;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 50px;
+  margin: 0px;
+  border: none;
+  padding: 5px;
+  display: grid;
+  height: fit-content;
```

Comment:
You have height: 50px; and height: fit-content; declared. Consider removing one depending on your layout needs.

Reply:
Yeah, I noticed that, too.  Agreed.